src/main/scala/zio/prelude/coherent/coherent.scala

```diff
@@ -32,11 +32,11 @@ object AssociativeEitherDeriveEqualInvariant {
     }
 }
 
-trait AssociativeEqual[A] extends Associative[A] with Equal[A]
```

Comment:
The rule I had been trying to maintain here was that we listed the type classes in alphabetical order in the coherent instances.

Reply:
:heavy_check_mark: